registry/api/v2/descriptors.go

```diff
@@ -514,7 +514,7 @@ var routeDescriptors = []RouteDescriptor{
 									digestHeader,
 								},
 								Body: BodyDescriptor{
-									ContentType: "application/json; charset=utf-8",
+									ContentType: "<media type of manifest>",
```

Comment:
This change is safe, right? I don't believe it's used except to generate the API spec, but it would be good to confirm.